test/lib/mayaUsd/fileio/testSchemaApiAdaptor.py

```diff
@@ -373,11 +373,11 @@ def testComplexAdaptation(self):
         cmds.setKeyframe(bulletPath, at="mass", t=10, v=30.0)
 
         # Modify the velocity so it can be exported to the RBD Physics schema.
-        cmds.setKeyframe(bulletPath, at="initialVelocityX", t=0, v=5.0)
+        cmds.setKeyframe(bulletPath, at="initialVelocityX", t=1, v=5.0)
         cmds.setKeyframe(bulletPath, at="initialVelocityX", t=10, v=50.0)
-        cmds.setKeyframe(bulletPath, at="initialVelocityY", t=0, v=6.0)
+        cmds.setKeyframe(bulletPath, at="initialVelocityY", t=1, v=6.0)
         cmds.setKeyframe(bulletPath, at="initialVelocityY", t=10, v=60.0)
-        cmds.setKeyframe(bulletPath, at="initialVelocityZ", t=0, v=7.0)
+        cmds.setKeyframe(bulletPath, at="initialVelocityZ", t=1, v=7.0)
```

Comment:
There is a test at line 435 that the exported value is (5,6,7) at t=1. If we really pull at the current time, this test can only succeed if the keyframe is a t=1. It used to pass, showing that the export was reading stale values. With the fix I need to keyframe correctly to get the right result.